src/env.cc

```diff
@@ -277,7 +277,8 @@ std::string GetExecPath(const std::vector<std::string>& argv) {
   uv_fs_t req;
   req.ptr = nullptr;
   if (0 ==
-      uv_fs_realpath(env->event_loop(), &req, exec_path.c_str(), nullptr)) {
+      uv_fs_realpath(Environment::GetThreadLocalEnv()->event_loop(),
```

Comment:
You can also use `nullptr` for the loop as this is a synchronous call, that might be easier